cabal-install/Distribution/Client/SetupWrapper.hs

```diff
@@ -256,7 +262,8 @@ type SetupMethod = Verbosity
 internalSetupMethod :: SetupMethod
 internalSetupMethod verbosity options _ bt mkargs = do
   let args = mkargs cabalVersion
-  debug verbosity $ "Using internal setup method with build-type " ++ show bt
+  --TODO: just temporarily higher verbosity
+  info verbosity $ "Using internal setup method with build-type " ++ show bt
```

Comment:
Ta. I'll eventually prepare patches by a manual diff process, so I'll notice things like this.